pkg/controllers/user/helm/common.go

```diff
@@ -118,36 +137,36 @@ func generateTemplates(obj *v3.App, templateVersionClient mgmtv3.CatalogTemplate
 		setValues = append([]string{"--set"}, strings.Join(result, ","))
 	}
 
-	commands := append([]string{"template", dir, "--name", obj.Name, "--namespace", obj.Spec.TargetNamespace}, setValues...)
+	commands := append([]string{"template", appDir, "--name", obj.Name, "--namespace", obj.Spec.TargetNamespace}, setValues...)
 
 	cmd := exec.Command(helmName, commands...)
 	sbOut := &bytes.Buffer{}
 	sbErr := &bytes.Buffer{}
 	cmd.Stdout = sbOut
 	cmd.Stderr = sbErr
 	if err := cmd.Start(); err != nil {
-		return "", "", "", errors.Wrapf(err, "helm template failed. %s", filterErrorMessage(sbErr.String(), dir, "template-dir"))
+		return "", "", "", tempDir, errors.Wrapf(err, "helm template failed. %s", sbErr.String())
```

Comment:
Did you intentionally remove filterErrorMessage? It was added to fix another problem to avoid random message being written into etcd.

Reply:
Was an accident, thanks for finding that!